src/applications/personalization/profile/components/connected-apps/AppDeletedAlert.jsx

```diff
@@ -10,8 +10,29 @@ export const AppDeletedAlert = props => {
     focusElement('[data-focus-target]');
   }, []);
 
-  const { id, title, dismissAlert } = props;
-  const alertMessage = `We’ve disconnected ${title}. This app can’t access any new information from your VA.gov profile. Some apps may still store information you’ve already shared. If you’d like to ask the app to delete any stored information, contact the app’s support.`;
+  const { id, title, dismissAlert, privacyUrl } = props;
+  const privacyLink = (
+    <a
+      href={privacyUrl}
+      target="_blank"
+      rel="noopener noreferrer"
+      aria-label={`${title} privacy policy`}
+    >
+      privacy policy
+    </a>
+  );
+  const alertMessage = (
+    <span>
+      We’ve disconnected {title}. This app can’t access any new information from
+      your VA.gov profile, but{' '}
+      <strong>
+        some apps may still store information you’ve already shared.
+      </strong>{' '}
+      To learn about how this app stores your information, including what
+      information it stores, if any, and what steps are available to you, review
+      the {title} {privacyLink}.
```

Comment:
Can we safely assume that all apps will always have a privacyLink? Just asking to make sure this copy will always make sense 😃

Reply:
Terms of service and privacy policy review are a part of the consumer onboarding process so it's safe to assume that apps will have a privacy link.